ci-operator/config/amq-streams-qe/strimzi-kafka-operator/OWNERS

```diff
@@ -0,0 +1,12 @@
+approvers:
+- ascerra
```

Comment:
We can remove anyone from cspi/interop and just use the `cspi-qe-ocp-lp` alias.

Reply:
ok, will do